src/packages/rlp/tests/dataTypes.test.ts

```diff
@@ -40,33 +40,17 @@ describe("RLP encoding (string):", function () {
   it("should return itself if single byte and less than 0x7f:", function () {
     const encodedSelf = RLP.encode(Buffer.from("a"));
     assert.strictEqual(encodedSelf.toString(), "a");
-    assert.strictEqual(RLP.getLength(encodedSelf), 1);
+    assert.strictEqual(encodedSelf.length, 1);
   });
 
   it("length of string 0-55 should return (0x80+len(data)) plus data", function () {
     const encodedDog = RLP.encode(Buffer.from("dog"));
     assert.strictEqual(4, encodedDog.length);
-    assert.strictEqual(RLP.getLength(encodedDog), 4);
     assert.strictEqual(encodedDog[0], 131);
     assert.strictEqual(encodedDog[1], 100);
     assert.strictEqual(encodedDog[2], 111);
     assert.strictEqual(encodedDog[3], 103);
   });
-
-  it("length of string >55 should return 0xb7+len(len(data)) plus len(data) plus data", function () {
-    const encodedLongString = RLP.encode(
-      Buffer.from(
-        "zoo255zoo255zzzzzzzzzzzzssssssssssssssssssssssssssssssssssssssssssssss"
-      )
-    );
-    assert.strictEqual(72, encodedLongString.length);
-    assert.strictEqual(RLP.getLength(encodedLongString), 2);
-    assert.strictEqual(encodedLongString[0], 184);
-    assert.strictEqual(encodedLongString[1], 70);
-    assert.strictEqual(encodedLongString[2], 122);
-    assert.strictEqual(encodedLongString[3], 111);
-    assert.strictEqual(encodedLongString[12], 53);
-  });
```

Comment:
Why is this test no longer valid?

Reply:
I actually don't know - the test passes just fine. @davidmurdoch you did most of the RLP work, do you know why I would have deleted this?

Reply:
> This test was testing `getLength` itself
That's not immediately clear to me, isn't this testing that RLP correctly handles longer strings?

Reply:
I did some digging and I take it back. I don't know why I removed this one!

Reply:
I've put it back here: https://github.com/trufflesuite/ganache/pull/3971/commits/295e95196efafddd880de43d374bd02632a315af

I hope it works